### DISCUSSION TITLE
Downloading a different branch

### DISCUSSION THREAD

**Discussion by BEisem** [Taxel/PlexTraktSync#217] [Q&A] [answered]
Is it possible to download the 0.7x branch that has been actively updated? Or do I have to wait for it to be merged into the master branch?

**Comment by glensc** [comment, marked as answer]
1. checkout whatever tag/branch you want/need: https://github.com/Taxel/PlexTraktSync/pull/195/files
2. project administrator @Taxel can't be reached, and the secondary developer is hesitant to approve https://github.com/Taxel/PlexTraktSync/pull/209 you can test that and give positive feedback for him to be more confident.

**Reply by BEisem** [reply]
I'm so sorry, I'm not following. I want to download and test your latest version that you want to have approved. Where can I do that? If I go to https://github.com/Taxel/PlexTraktSync/pull/209 it just shows me a bunch of changes you have made, but I don't see where to download the package.

**Reply by glensc** [reply]
@BEisem 

1. I pointed your documentation pull request that explains how to check out the wanted version. read the proposed changes. it explains how
2. you can download. need two clicks. click on source branch and then download from there

![image](https://user-images.githubusercontent.com/199095/114281051-da8c1800-9a44-11eb-88e6-adc996747d8d.png)



![image](https://user-images.githubusercontent.com/199095/114281081-fb546d80-9a44-11eb-9e35-131e8266c5b8.png)
